packages/@coorpacademy-components/src/organism/filter-checkbox-and-search/index.tsx

```diff
@@ -0,0 +1,111 @@
+import React, {useCallback, useState} from 'react';
+import {filter, pipe, size, toString} from 'lodash/fp';
+import Title from '../../atom/title';
+import Tag from '../../atom/tag';
+import ButtonLink from '../../atom/button-link';
+import SearchForm from '../../molecule/search-form';
+import CmCheckboxWithText from '../../molecule/cm-checkbox-with-text';
+
+import Provider, {GetTranslateFromContext} from '../../atom/provider';
+import {WebContextValues} from '../../atom/provider/web-context';
+import style from './style.css';
+import propTypes, {FilterCheckboxAndSearchProps} from './props-types';
+
+const clearButtonStyle = {fontWeight: 'normal', position: 'absolute', right: 0};
+const showButtonStyle = {
+  fontSize: '14px',
+  fontWeight: '600',
+  fontFamily: 'Gilroy',
+  marginTop: '16px'
+};
+const INITIAL_VISIBLE_OPTIONS = 5;
+
+const FilterCkeckboxAndSearch = (
+  props: FilterCheckboxAndSearchProps,
+  context: WebContextValues
+) => {
+  const {title, placeholder, withSearch, onClear, options} = props;
+  const translate = GetTranslateFromContext(context);
+  const [searchValue, setSearchValue] = useState('');
+  const onSearchChange = useCallback(value => {
+    setSearchValue(value);
+  }, []);
+  const [showMore, setShowMore] = useState(false);
+  const selectedFiltersCount = pipe(filter({selected: true}), size)(options);
+  const hasSelectedFilters = selectedFiltersCount > 0;
+  const handleShowMore = useCallback(() => {
+    setShowMore(!showMore);
+  }, [showMore]);
+  const visibleOptions = showMore ? options : options.slice(0, INITIAL_VISIBLE_OPTIONS);
+  return (
+    <div className={style.container} data-testid="filter-checkbox-and-search-container">
+      <div className={style.header} data-testid="header">
+        <div className={style.titleAndTagWrapper} data-testid="title-and-tag-wrapper">
+          <Title title={title} />
+          {hasSelectedFilters ? (
+            <Tag label={toString(selectedFiltersCount)} type="info" size="S" />
+          ) : null}
+        </div>
+        {hasSelectedFilters ? (
+          <div className={style.clearButtonContainer}>
+            <ButtonLink
+              label={'Clear'}
+              type="text"
+              data-testid="clear-button-link"
+              onClick={onClear}
+              customStyle={clearButtonStyle}
+            />
+          </div>
+        ) : null}
+      </div>
+      {withSearch ? (
+        <div className={style.search} data-testid="search-container">
+          <SearchForm
+            search={{
+              placeholder,
+              value: searchValue,
+              onChange: onSearchChange
+            }}
+          />
+        </div>
+      ) : null}
+      <div data-testid="options-container" className={style.optionsContainer}>
+        {visibleOptions.map(option => (
```

Comment:
```suggestion
        {visibleOptions.map(({value, label, selected}) => (
```

pour éviter de ré écrire option à chaque fois